swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, condition = None, age=None, name = None, fabric = None):
+        super().__init__(id, condition, age, name = "Clothing")
```

Comment:
Nice work with the `super` function, before using this we know that to get the same behavior we would have to write a bunch of the same code, which doesn't fall in line with our `DRY` paradigm, this saves us a lot of typing and causes the code base to much more concise. Also, great work on passing in the `None` default param in the `__init__` function.